kubespawner/spawner.py

```diff
@@ -1437,67 +1437,68 @@ def _validate_image_pull_secrets(self, proposal):
     profile_form_template = Unicode(
         """
         <style>
-        /* The profile description should not be bold, even though it is inside the <label> tag */
-        #kubespawner-profiles-list .profile p, #kubespawner-profiles-list .profile label {
-            font-weight: normal;
-        }
-
-        #kubespawner-profiles-list .profile {
-            display: flex;
-            flex-direction: row;
-            border-bottom: 1px solid #ccc;
-            padding-bottom: 12px;
-        }
+            /*
+                .profile divs holds two div tags: one for a radio button, and one
+                for the profile's content.
+            */
+            #kubespawner-profiles-list .profile {
+                display: flex;
+                flex-direction: row;
+                font-weight: normal;
+                border-bottom: 1px solid #ccc;
+                padding-bottom: 12px;
+            }
 
-        #kubespawner-profiles-list .profile .check {
-            display: flex;
-            align-content: center;
-            padding: 12px;
-        }
+            #kubespawner-profiles-list .profile .radio {
+                padding: 12px;
+            }
```

Comment:
I renamed "check" to "radio" as it got me confused it was named "check" as in a checkbox compared to a radiobox we actually use.